src/section.js

```diff
@@ -88,16 +88,9 @@ class Section {
 
 		this.load(_request).
 			then(function(contents){
-				var serializer;
-				var Serializer;
-
-				if (typeof XMLSerializer === "undefined") {
-					Serializer = require("xmldom").XMLSerializer;
-				} else {
-					Serializer = XMLSerializer;
```

Comment:
Ah ok, let just do some browser detection for IE here with something like
```
var userAgent = (typeof navigator !== 'undefined' && navigator.userAgent) || '';
var isIE = userAgent.indexOf('Trident') >= 0;

if (typeof XMLSerializer === "undefined" || isIE) {
	Serializer = require("xmldom").XMLSerializer;
} else {
	Serializer = XMLSerializer;
}
```

and I will include xmldom in the webpack for the epub.legacy file